buddyboss/buddyboss-platform
# Image shared on the group page through the site-wide activity page using the "Post In:" setting are not showing up on the group page > photos section

### All Submissions:






### Changes proposed in this Pull Request:



Closes #397 .

### How to test the changes in this Pull Request:

1. Go to 'Site-wide activity page'
2. Click on 'Post In' and choose the group where you want to post the image
3. Go to the group's page
4. Open Photos tab, you won't see the image there

### Proof Screenshots or Video
https://www.loom.com/share/da9d3b19b73e4d5695145f5d38fb5563

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
